soundcraft/constants.py

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (c) 2020 Jim Ramsay <i.am@jimramsay.com>
+# Copyright (c) 2020,2021 Hans Ulrich Niedermann <hun@n-dimensional.de>
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+########################################################################
+#
+# This file is intended to work without importing any non-standard
+# Python libraries; ideally without any imports at all.
+#
+# This allows us to import this both from the setuptools `setup.py`
+# and from any code inside `soundcraft/` as well.
+#
+########################################################################
+
+# Package name
+PACKAGE = "soundcraft-utils"
+VERSION = "0.4.90"
```

Comment:
Not important for this commit per-se, but I think this degree of change warrants a bump to 1.0

Reply:
I think we're about halfway on the way to 1.0, so 0.5 would be the proper next release number.

Reply:
Bumping this to `0.4.91` after merging the `session-dbus` and to `0.4.92` after merging the Great Rename could be useful.